Gemfile

```diff
@@ -18,7 +20,7 @@ group :development, :test do
   gem "sqlite3" # Remove this when you choose a production database
   gem "factory_girl_rails", "~> 4.7"
   gem "timecop"
-  gem "webmock", require: false
+  gem "webmock"
```

Comment:
ah just saw the last file - where that is changed to "webmock/rspec" any reason why we've done that? I don't know pros and cons just noticed it's different to Pub API